emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -461,8 +461,6 @@ def find_entries_count(self, key_list = None, time_query = None, geo_query = Non
 
         For key_list = None or empty, total count of all documents are returned considering the matching entries from entire dataset.
         """
-        print("builtin_timeseries.find_entries_count() called")
-        
```

Comment:
why did you remove this?

Reply:
I had these print statements while working on their respective issues. I had originally meant to use them just while testing but forgot to remove them. Hence thought of cleaning the output statements generated.